### ISSUE TITLE
Slow Zoom

### ISSUE THREAD

**Issue by benohb** [LibreCAD/LibreCAD#965]
### Expected behavior
What is expected
### Observed behavior
The hard drive operates at every click when using the spline or/and zooming 
Something that is written every time (zooming) to a file /home/user/.config/LibreCAD/LibreCAD.conf
 Make this file read-only  ... problem disappears
### Steps to reproduce or sample file
- New file 
- Spline  
- mouse wheel / click
### Operating System and LibreCAD version info
Linux debian 64 
libc6 2.27

**Comment by benohb** [comment]
```
 /sudo fatrace 
librecad(20964): CWO /home/hb/.config/LibreCAD/LibreCAD.conf
librecad(20964): O /home/hb/.config/LibreCAD/LibreCAD.conf.lock
librecad(20964): W /home/hb/.config/LibreCAD/LibreCAD.conf.lock
librecad(20964): O /home/hb/.config/LibreCAD/LibreCAD.conf.Z20964
librecad(20964): W /home/hb/.config/LibreCAD/LibreCAD.conf.Z20964
librecad(20964): CW /home/hb/.config/LibreCAD/LibreCAD.conf
librecad(20964): CW /home/hb/.config/LibreCAD/LibreCAD.conf.lock
librecad(20964): CWO /home/hb/.config/LibreCAD/LibreCAD.conf
librecad(20964): WO /home/hb/.config/LibreCAD/LibreCAD.conf.lock
librecad(20964): O /home/hb/.config/LibreCAD/LibreCAD.conf.B20964
librecad(20964): W /home/hb/.config/LibreCAD/LibreCAD.conf.B20964
librecad(20964): CW /home/hb/.config/LibreCAD/LibreCAD.conf
librecad(20964): CW /home/hb/.config/LibreCAD/LibreCAD.conf.lock (deleted)
```

**Comment by mpiffault** [comment]
#859

**Comment by mpiffault** [comment]
This due to LibreCad writing current tool options to disk on every action, including zooming. See #859 and fix #1079 .
